crates/ruff_db/src/files.rs

```diff
@@ -290,20 +274,23 @@ pub struct File {
 
     /// The unix permissions of the file. Only supported on unix systems. Always `None` on Windows
     /// or when the file has been deleted.
+    #[default]
```

Comment:
For my education: what does the `#[default]` attribute do here?

Reply:
Will see if it gets accepted. It initializes the value with `Default::default` unless it's specified explicilty